WordPress/WordPressTest/PostBuilder.swift

```diff
@@ -207,21 +207,4 @@ class PostBuilder {
         // assert(post.managedObjectContext != nil)
         return post
     }
-
-    static func setUpInMemoryManagedObjectContext() -> NSManagedObjectContext {
-        let managedObjectModel = NSManagedObjectModel.mergedModel(from: [Bundle.main])!
-
-        let persistentStoreCoordinator = NSPersistentStoreCoordinator(managedObjectModel: managedObjectModel)
-
-        do {
-            try persistentStoreCoordinator.addPersistentStore(ofType: NSInMemoryStoreType, configurationName: nil, at: nil, options: nil)
-        } catch {
-            print("Adding in-memory persistent store failed")
-        }
-
-        let managedObjectContext = NSManagedObjectContext(concurrencyType: .mainQueueConcurrencyType)
-        managedObjectContext.persistentStoreCoordinator = persistentStoreCoordinator
-
-        return managedObjectContext
-    }
```

Comment:
This is redundant when we already set up an in-memory store in `CoreDataTestCase` via this utility:

https://github.com/wordpress-mobile/WordPress-iOS/blob/21274d482e58bd0f18f8af991b7e6878c6258e4a/WordPress/WordPressTest/ContextManager%2BHelpers.swift#L7-L12